test/config-luatest/rpc_test.lua

```diff
@@ -16,7 +16,7 @@ local function skip_if_no_vshard()
 end
 
 g.test_connect = function(g)
-    local dir = treegen.prepare_directory({}, {})
+    local dir = treegen.prepare_directory(g, {}, {})
```

Comment:
It seems it is an artifact of rebasing after bd27df009c403e89c003d5b66763c0f0bbf08440.

Reply:
Fixed. Thank you.